src/components/NavBar/NavBar.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import NavBar from './NavBar.component';
+
+describe('<NavBar /> component', () => {
+  let component;
```

Comment:
you can use `render(<NavBar />);` and then use `screen.queryBy...` to access the dom, to avoid declaring this variable